src/reaper_osara.cpp

```diff
@@ -2055,6 +2055,47 @@ void cmdShowContextMenu3(Command* command) {
 	showReaperContextMenu(2);
 }
 
+void cmdReportAutomationMode(Command* command) {
+	// This reports the global automation override if set, otherwise tcurrent track automation mode.
+	ostringstream s;
+	MediaTrack* track = GetLastTouchedTrack();
+	if (GetGlobalAutomationOverride() >= 0) {
+		s << "global automation override " << automationModeAsString(GetGlobalAutomationOverride());
+	} else {
+		s << "track automation mode " << automationModeAsString(GetTrackAutomationMode(track));
+	}
+	outputMessage(s);
+}
+
+void cmdToggleGlobalAutomationLatchPreview(Command* command) {
+	if (GetGlobalAutomationOverride() == 5) {  // in latch preview mode
+		SetGlobalAutomationOverride(-1);
+		outputMessage("Global automation override off");
+	}
+	else { //not in latch preview.
```

Comment:
nit: Please put the else on the same line as the closing brace for the if; i.e. `} else {`